pkg/services/accesscontrol/resourcepermissions/store.go

```diff
@@ -582,11 +556,25 @@ func (s *store) getOrCreateManagedRole(sess *sqlstore.DBSession, orgID int64, na
 	return &role, nil
 }
 
-func (s *store) getResourcePermissionsByIds(sess *sqlstore.DBSession, resource, resourceID, resourceAttribute string, ids []int64) ([]flatResourcePermission, error) {
-	var result []flatResourcePermission
-	if len(ids) == 0 {
-		return result, nil
+func generateNewRoleUID(sess *sqlstore.DBSession, orgID int64) (string, error) {
+	for i := 0; i < 3; i++ {
```

Comment:
we should rethink of a better way for this one day

Reply:
Yeah...